src/mpc/mpc_common.hpp

```diff
@@ -1,20 +1,30 @@
+#ifndef __ZETH_MPC_MPC_COMMON_HPP__
+#define __ZETH_MPC_MPC_COMMON_HPP__
+
+#include "include_libsnark.hpp"
+
 #include <boost/program_options.hpp>
 #include <fstream>
 #include <string>
 #include <vector>
 
+using ppT = libff::default_ec_pp;
+using FieldT = libff::Fr<ppT>;
```

Comment:
This defines the types for all the files in the mpc command, so are you suggesting we define it across the entire project? (it's currently defined in some of the snark headers, and util_api headers, as well as some .cpp files)

I certainly think it would be good to have a better way or place to define this if we find the right way.  It should be consistent across all executables, but doesn't necessarily need to be part of the zeth library itself.  Or maybe we should have configuration param for it like we have for SNARK_*?.

It wasn't immediately obvious where the right place was, so I made a single definition per executable, but I do think it would make sense to sort this out.  Let me know what you think.  Perhaps it needs more discussion.

Reply:
> This defines the types for all the files in the mpc command, so are you suggesting we define it across the entire project?

Not really. I think it's fine to define the types for each entrypoints. i was just suggesting to move the template instantiation in the same file for this entrypoint.
You used `using` directives in the `mpc_common.hpp` and in the `mpc_common.cpp` here.

Reply:
Ah, I see.  Thanks for the clarification.